pkg/watcher/fetcher.go

```diff
@@ -19,7 +19,7 @@ package watcher
 // Interface to be implemented by any metrics provider client to interact with Watcher
 type FetcherClient interface {
 	// Fetch metrics for given host
-	FetchHostMetrics(host string, window *Window) ([]Metric, error)
+	// FetchHostMetrics(host string, window *Window) ([]Metric, error)
```

Comment:
You don't have to comment this out if Prometheus client has no use cases at first. You can just return an error in implemented function along the lines of "This API is not supported".

Reply:
Ok, will revise accordingly.